.travis.yml

```diff
@@ -1,24 +1,24 @@
 language: php
 
 php:
-    - 7.1
+- 7.1
 
 cache:
     directories:
-        - vendor
-        - $HOME/.composer/cache
+    - vendor
+    - $HOME/.composer/cache
 
 env:
     global:
-        - APPLICATION_ENV=development
-        - APPLICATION_STORE=DE
+    - APPLICATION_ENV=development
+    - APPLICATION_STORE=DE
 
 install:
-    - composer install --no-interaction --prefer-dist
+- composer install --no-interaction --prefer-dist
 
 script:
-    - vendor/bin/phpcs src --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml
-    - vendor/bin/phpmd src text vendor/spryker/architecture-sniffer/src/ruleset.xml --minimumpriority=2
+- sh -c "if [ '$DEFAULT' = '1' ]; then vendor/bin/codecept run; fi"
+- sh -c "if [ '$PHPCS' = '1' ]; then composer cs-check ; fi"
```

Comment:
This is never run! Make sure you also include the scripts part in composer.json here and make the matrix here appear for it.